### PR TITLE
end of month balance freeze

### PR DESCRIPTION
**Pull request by TiviPlus**
did you think I would forget

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,13 +1,4 @@
...
-
-## Reporting Issues
-
-See [this page](https://github.com/tgstation/TerraGov-Marine-Corps/issues/new?template=bug_report.md) for a guide and format to issue reports.
-
-## Introduction
-
-Hello and welcome to the TGMC contributing page. You are here because you are curious or interested in contributing - thank you! Everyone is free to contribute to this project as long as they follow the simple guidelines and specifications below. We strive to maintain long-term code stability and maintainability, and to do that, we need all pull requests to hold up to those specifications. It's in everyone's best interests - including yours! - if the same bug doesn't have to be fixed twice because of duplicated code.
-
+ndsfpihgrph4etr[0hgkojp
```
Comment (Guidesu) [inline .github/CONTRIBUTING.md:1]: oie YHIF 9SOEDGHFDIOHGFDHFGHJG

**Comment by neohenry** [comment]
Love to see this again. It was great to see this for an entire month, and a lot of clean up happened. Hopefully we can recreate the magic!

**Comment by David-Stormwell** [comment]
how am I supposed to pr the code I have to change the entire game from the ground up now?!??

**Comment by Tennessee116** [comment]
The one thing holding @Pariah919 at bay from reaching -200 fix/feature pull request delta.

**Pull request closed without merging**